lib/govuk_publishing_components/presenters/content_breadcrumbs_based_on_priority.rb

```diff
@@ -74,17 +72,34 @@ def priority_taxon?(taxon)
         PRIORITY_TAXONS.values.include?(taxon["content_id"])
       end
 
+      def brexit_child_taxon?(taxon)
+        brexit_child_taxons.include?(taxon["content_id"])
+      end
+
+      def brexit_child_taxons
+        [PRIORITY_TAXONS[:brexit_business], PRIORITY_TAXONS[:brexit_individuals]]
+      end
+
       def preferred_priority_taxon
         query_parameters["priority-taxon"] if query_parameters
       end
 
       def tracking_action
         action = %w[superBreadcrumb]
-        action << "Brexitbusiness" if taxon["content_id"] == BREXIT_TAXONS[:brexit_business]
-        action << "Brexitcitizen" if taxon["content_id"] == BREXIT_TAXONS[:brexit_business]
-        action << "Brexitbusinessandcitizen" if taxon["content_id"] == BREXIT_TAXONS[:brexit]
+        action << "Brexitbusiness" if taxon["content_id"] == PRIORITY_TAXONS[:brexit_business]
+        action << "Brexitcitizen" if taxon["content_id"] == PRIORITY_TAXONS[:brexit_individuals]
+        action << "Brexitbusinessandcitizen" if taxon["content_id"] == PRIORITY_TAXONS[:brexit_taxon]
         action.join(" ")
       end
+
+      def tagged_to_both_brexit_child_taxons?
+        t = priority_taxons.select { |taxon| brexit_child_taxon?(taxon) }
```

Comment:
I was wondering whether we should do a `uniq` here to guarantee that we haven't somehow picked up duplicate tags to one of the child taxons, but I think it's OK. (I was dreaming of a potential future with a deeper taxon structure)

Reply:
Yes good idea, I'll do that.